web/src/components/messages/status-texts.tsx

```diff
@@ -0,0 +1,51 @@
+import { Text, TextVariants } from '@patternfly/react-core';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Status } from '../../api/loki';
+
+export interface StatusProps {
+  status: Status;
+}
+
+export const StatusTexts: React.FC<StatusProps> = ({ status }) => {
+  const { t } = useTranslation('plugin__netobserv-plugin');
+
+  return (
+    <>
+      {status && status.isLokiReady === false && (
```

Comment:
I guess isAllowLoki is missing here (I'm seeing this error while Loki is disabled)
```suggestion
      {status && status.isAllowLoki && status.isLokiReady === false && (
```